_docs/_latest/configure/scanner/rules.md

```diff
@@ -365,35 +365,58 @@ rules:
     * **Description**: A list of one or more resource ids to match.
     * **Valid values**: String, you can use `*` to match for all.
 
-* `dataset_id`
-  * **Description**: The BigQuery dataset to which you want to apply the rule.
+* `dataset_ids`
+  * **Description**: List of BigQuery datasets to which you want to apply the rule.
   * **Valid values**: String, you can use `*` to match for all.
 
+* `bindings`
+  * **Description**: The BigQuery ACL rule bindings to bind members to a role.
+    * `role`
+      * **Description**: A [BigQuery ACL role](https://cloud.google.com/storage/docs/access-control/lists).
+      * **Valid values**: One of `OWNER`, `WRITER` or `READER`.
+    * `members`
+      * **Description**: A list of members. You can also use an empty list. Only a single field must be set per member.
+        * `domain` 
+          * **Description**: Domain.
+          *  **Valid values**: String.
+        * `group_email`
+          * **Description**: Group email.
+          * **Valid values**: String.
+        * `user_email`
+          * **Description**: User email.
+          * **Valid values**: String.
+        * `special_group`
+          * **Description**: Special group.
+          * **Valid values**: String.
+
 * `special_group`
-  * **Description**: The special group.
+  * **Description**: The special group. ***DEPRECATED, prefer to use bindings.***
```

Comment:
What do you think about "DEPRECATED, please use bindings instead" ?

Reply:
Done.